README.md

```diff
@@ -1,62 +1,181 @@
 # Exercism v3 Local Environment
 
-This provides a local environment for running Exercism on Docker. It is very much a WIP.
+Hello 👋
 
-## Getting Started
+This repository will provide you with a local setup of Exercise that you can use to develop any part of the project.
 
-Run the following line-by-line.
+It is built using [Docker](https://www.docker.com/), which you need to install, but it does not require any real Docker knowledge to use.
+
+Our aim is to get you to a working setup within 10-minutes from now (presuming you have a decent internet connection).
+It's also possible to get each piece of Exercism working on your local machine without this repository, but that will take lots more effort and work.
+
+## Basic setup
+
+The following instructions take you through getting the most basic setup working.
+We'll explain what actually happens below the hood, and how to configure things afterwards
+
+### Getting set up
+
+To start, run the following instructions line-by-line:
 
 ```bash
 # Make a directory to host all of Exercism within
 mkdir exercism
 
-# cd into it
+# Move into the new directory
 cd exercism
 
-# Clone this repo
+# Clone this repository onto your computer
 git clone git@github.com:exercism/v3-docker-compose.git
```

Comment:
Assumes `git cli` is installed. Should probably be noted up top.

Reply:
Added above :+1: